backend/app/assets/stylesheets/spree/backend/sections/_orders.scss

```diff
@@ -49,6 +49,14 @@
   margin-bottom: 18px;
 }
 
+[data-hook="complete-order-button"] {
+  margin-bottom: 15px;
```

Comment:
I don't see this on the other sections. It might be present and I just haven't noticed it, but I don't think we should add bottom padding to the whole admin. My sandbox is currently broken because of #2989, but I will look for more instances of this.